src/platforms/native/common/debug-information.mdx

```diff
@@ -14,11 +14,30 @@ libraries. If you are not sure which files are required, go to _Project
 Settings > Processing Issues_, which shows a list of all the necessary files and
 instructions to retrieve them.
 
+<PlatformSection notSupported={["native.wasm"]}>
```

Comment:
Add empty lines (also at the closing tag):

```suggestion
<PlatformSection notSupported={["native.wasm"]}>

```

Otherwise it doesn't parse as Markdown and renders like this:

![image](https://user-images.githubusercontent.com/1433023/100853675-53240c00-3488-11eb-9739-e86a93b3e12c.png)